README.md

```diff
@@ -4,7 +4,9 @@
 # CARMASimulation
 This repository will host the CARMA Everything-in-the-Loop Co-Simulation tool. This XiL simulation platform will support CARLA and SUMO simulation environments as a co-simulation using the MOSAIC framework to facilitate coordination and data exchange. This co-simulation environment will also utilize NS-3 to simulate the communications used by the C-ADS systems.
 
-The current version and release history can be found here: [Release Notes](<docs/Release_notes.md>)
+## Release Notes
+The current version and release history of the CARMA software platform: [CARMA Release Notes](<docs/Release_notes.md>)
```

Comment:
This url still goes to cda sim's old release notes, please update to point to the platform's release notes